SwiftCompilerSources/Sources/Optimizer/PassManager/Context.swift

```diff
@@ -256,6 +256,22 @@ struct FunctionPassContext : MutatingContext {
     return false
   }
 
+  func specializeVTable(for type: Type, in function: Function) -> VTable? {
+    guard let vtablePtr = _bridged.specializeVTableForType(type.bridged, function.bridged) else {
+      return nil
+    }
+    notifyInstructionsChanged()
```

Comment:
`notifyInstructionsChanged` is not needed, because not instructions in the _current_ function are changed. But it doesn't harm.

Reply:
Fixed